File: Backend/Core/src/main/java/com/ssafy/pickit/domain/auth/dto/LoginResponse.java
package com.ssafy.pickit.domain.auth.dto;

public record LoginResponse(
	Long memberId,
	Boolean isExist,
	String socialId,
	String accessToken,
	String refreshToken
) {
	public static LoginResponse of(Long memberId, Boolean isExist, String socialId, String accessToken,
		String refreshToken) {
		return new LoginResponse(memberId, isExist, socialId, accessToken, refreshToken);
	}
}

File: Android/app/src/main/java/com/ssafy/pickit/data/datasource/remote/response/Auth/LoginResponse.kt
package com.ssafy.pickit.data.datasource.remote.response.Auth

import com.google.gson.annotations.SerializedName

data class LoginResponse(
    @SerializedName("is_exist")
    val isExist: Boolean,
    @SerializedName("social_id")
    val socialId: String?,
    @SerializedName("access_token")
    val accessToken: String?,
    @SerializedName("refresh_token")
    val refreshToken: String?
)